source.php

```diff
@@ -760,20 +646,17 @@ function rtsocial( $args = array() ) {
 	// G+ Share Button.
 	if ( ! empty( $options )
 	&& ! empty( $options['active'] )
-	&& in_array( 'gplus', $options['active'], true )
-	&& ! empty( $options['google_api_key'] ) ) {
+	&& in_array( 'gplus', $options['active'], true ) ) {
 		$gplus = array_search( 'gplus', $options['active'], true );
 
-		$gplus_count = ( empty( $options['hide_count'] ) || 1 !== (int) $options['hide_count'] ) ? '<div class="rtsocial-' . $options['display_options_set'] . '-count"><div class="rtsocial-' . $options['display_options_set'] . '-notch"></div><span class="rtsocial-gplus-count"></span></div>' : '';
-
 		$gplus_layout = '<div class="rtsocial-gplus-' . $options['display_options_set'] . '">';
 
 		if ( 'horizontal' === $options['display_options_set'] ) {
-			$gplus_layout .= '<div class="rtsocial-gplus-' . $options['display_options_set'] . '-button"><a class="rtsocial-gplus-button" href= "https://plus.google.com/share?url=' . $rtslink . '" rel="nofollow" target="_blank" title="+1: ' . $rtatitle . '"></a></div>' . $gplus_count;
```

Comment:
Use `sprintf` instead of concatenation.
It will be easier to find the things right.